scripts/dockerDeleteExpiredAccessTokens.sh

```diff
@@ -26,5 +27,9 @@ log "Delete expired access tokens: starting..."
 log "CouchDB: ${GPII_COUCHDB_URL_SANITIZED}"
 log "Delete all flag: ${DELETE_ALL}"
 
-node ${GPII_APP_DIR}/scripts/deleteExpiredAccessTokens.js ${GPII_COUCHDB_URL} ${DELETE_ALL}
-
+node ${DELETE_ACCESS_TOKENS_JS} ${GPII_COUCHDB_URL} ${DELETE_ALL}
+err=$?
+if [ "${err}" != '0' ]; then
+  log "${DELETE_ACCESS_TOKENS_JS} failed with ${err}, exiting"
```

Comment:
It might be easier to understand by saying "failed with the exit code ${err}" to indicate what this number is.

Reply:
Okay.  I also got rid of the trailing ",  exiting".  It's redundant.